Formula/telegraf.rb

```diff
@@ -392,6 +393,8 @@ def plist; <<-EOS.undent
           <string>#{opt_bin}/telegraf</string>
           <string>-config</string>
           <string>#{HOMEBREW_PREFIX}/etc/telegraf.conf</string>
+          <string>-config-directory</string>
+          <string>#{HOMEBREW_PREFIX}/etc/telegraf.d</string>
```

Comment:
Aware this is on the above line as well, but `{etc}` == `#{HOMEBREW_PREFIX}/etc`